src/run.rs

```diff
@@ -19,6 +19,7 @@ pub(crate) async fn pull_and_run(
     request: &str,
     settings: Option<String>,
     verified_manifest_digest: &Option<String>,
+    sigstore_opts: &SigstoreOpts,
```

Comment:
clippy is complaining about the number of args we are passing. Maybe this is something we can fix by aggregating some of them into ad-hoc structs?

Reply:
I already did on this argument (it's a single struct containing two arguments). I also didn't want to block longer than needed for that refactor. I think the argument refactoring can be done later.

If you agree with that I'm taking care of the refactor, but we'd have to merge with a red clippy. I just didn't want to get into any parameter refactoring on this PR, that we are waiting for.

Reply:
Yes, that makes total sense to me!